src/_scss/pages/covid19/index.scss

```diff
@@ -1,343 +1,365 @@
-                    }
-                }
-                .covid__date-note {
-                    @include display(flex);
-                    width: 100%;
-                    justify-content: left;
-                    font-size: 1.2rem;
-                    font-style: italic;
-                    letter-spacing: 0;
-                    line-height: 1.5rem;
-                    margin-left: rem(5);
-                    @media(min-width: $medium-screen) {
-                        margin-left: rem(15);
-                    }
-                }
-                &.award_question {
-                    .body__header {
-                        display: none;
-                    }
-                    .award-question__container {
-                        @include display(flex);
-                        flex-wrap: wrap;
-                        text-align: center;
-                        width: 100%;
-                        .information-top {
-                            height: 0.5rem;
-                            width: 100%;
-                            border-radius: 0;
-                        }
-                        .glossary-term {
-                            font-weight: $font-semibold;
-                        }
-                        .award-question__title {
-                            margin: 0;
-                            font-size: 3.6rem;
-                            font-weight: $font-semibold;
-                            letter-spacing: 0;
-                            line-height: 4.5rem;
-                            text-align: center;
-                            padding: 6.2rem 2.5% 3.1rem;
-                            @media(min-width: $medium-screen) {
-                                padding: 6.2rem 15% 3.1rem;
-                            }
-                            width: 100%;
-                        }
-                        .award-question__sub-section {
-                            font-size: 1.4rem;
-                            letter-spacing: 0;
-                            line-height: 2.2rem;
-                            text-align: center;
-                            padding: 0 2.5%;
-                            @media(min-width: $medium-screen) {
-                                padding: 0 15% 6.1rem;
-                            }
-                            .award-question__sub-section_paragraph:first-of-type {
-                                margin-top: 0;
-                            }
-                        }
-                    }
-                }
-            }
-            .body__section:first-of-type {
-                padding: 0;
-            }
+      .body__section:last-child {
+        margin-bottom: 0;
+        padding: 0;
+      }
+    }
+  }
+  .footer-reference {
+    width: 100%;
+  }
+  @import "../../layouts/default/header/warning";
+}
 
-            .body__section:nth-child(4) {
-                padding: 0;
-            }
+.award_spending_by_recipient {
```

Comment:
The styling looks good. Could we move this block to the file that is scoped to the recipient section? `pages/covid19/recipient/recipientContainer.scss`?

Reply:
Tried that, it doesn't seem to be used